framework/src/interfaces/PerfGraphInterface.C

```diff
@@ -0,0 +1,43 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "PerfGraphInterface.h"
+
+#include "MooseApp.h"
+
+template <>
+InputParameters
+validParams<PerfGraphInterface>()
+{
+  InputParameters params = emptyInputParameters();
+  return params;
+}
+
+PerfGraphInterface::PerfGraphInterface(const MooseObject * moose_object, std::string prefix)
+  : _pg_params(&moose_object->parameters()),
+    _perf_graph(_pg_params->get<MooseApp *>("_moose_app")->perfGraph()),
```

Comment:
You'll want to use the `getChecedPointer` method instead, it'll prevent a weird error if this ever gets refactored or constructed with a `nullptr`.

Reply:
I guess I didn't here because I don't have access to the "normal" one in MooseObject... but I didn't think about the fact that I could, of course, just call it directly... 🤦‍♂️